app/models/event_user.rb

```diff
@@ -1,6 +1,6 @@
 class EventUser < ActiveRecord::Base
   # TODO Do we need these roles?
-  ROLES = [['Speaker', 'speaker'], ['Submitter', 'submitter'], ['Moderator', 'moderator']]
+  ROLES = [%w[Speaker speaker], %w[Submitter submitter], %w[Moderator moderator]]
```

Comment:
Yeah, it is confusing for me too. I think we should not enable this cop at all, and let people write things either way.